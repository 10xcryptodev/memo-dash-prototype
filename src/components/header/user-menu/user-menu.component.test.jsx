import React from 'react'
import { shallow } from 'enzyme'
import UserMenuComponent from './user-menu.component'

describe('<UserMenuComponent />', () => {
  let wrapper

  beforeEach(() => {
    const div = document.createElement('div')
    document.body.appendChild(div)
    wrapper = shallow(
      <UserMenuComponent
        username="Alice"
        avatar="Avatar"
        location="/"
        onSignOutClicked={jest.fn()}
        onProfileClicked={jest.fn()}
      />
    )
  })

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
