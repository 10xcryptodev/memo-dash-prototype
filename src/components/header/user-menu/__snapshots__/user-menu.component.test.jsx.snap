// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserMenuComponent /> renders without crashing 1`] = `
<Dropdown
  additionLabel="Add "
  additionPosition="top"
  closeOnBlur={true}
  deburr={false}
  icon="dropdown"
  item={true}
  minCharacters={1}
  noResultsMessage="No results found."
  openOnFocus={true}
  renderLabel={[Function]}
  searchInput="text"
  selectOnBlur={true}
  selectOnNavigation={true}
  simple={true}
  trigger={
    <span>
      <Image
        as="img"
        circular={true}
        size="mini"
        spaced="right"
        src="avatarUrl"
        ui={true}
      />
      Hello, 
      Alice
    </span>
  }
  wrapSelection={true}
>
  <DropdownMenu>
    <DropdownItem
      active={false}
      onClick={[Function]}
    >
      Your Profile
    </DropdownItem>
    <DropdownDivider />
    <DropdownItem
      onClick={[MockFunction]}
    >
      Sign Out
    </DropdownItem>
  </DropdownMenu>
</Dropdown>
`;
